Description: Remove the badge in the readme to avoid lintian complaining about
 privacy-breach-generic
Author: Joseph Herlant <herlantj@gmail.com>
Forwarded: not-needed
Last-Update: 2018-04-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/README.adoc
+++ b/README.adoc
@@ -412,7 +412,6 @@
 
 Discussion list (Nabble):: {uri-discuss}
 Twitter:: https://twitter.com/search?f=tweets&q=%23asciidoctor[#asciidoctor] hashtag or https://twitter.com/asciidoctor[@asciidoctor] mention
-Chat (Gitter):: image:https://badges.gitter.im/Join%20In.svg[Gitter, link=https://gitter.im/asciidoctor/asciidoctor]
 ////
 Chat (IRC):: {uri-irc}[#asciidoctor] on FreeNode IRC
 ////
